data_validation/__main__.py

```diff
@@ -131,12 +131,20 @@ def get_calculated_config(args, config_manager):
         col_list = None if args.hash == "*" else cli_tools.get_arg_list(args.hash)
         fields = config_manager._build_dependent_aliases("hash", col_list)
         aliases = [field["name"] for field in fields]
-
+        config_manager.append_dependent_aliases(aliases)
```

Comment:
We only need to append dependent aliases if a list of columns is provided. If someone runs `--hash '*'`, they don't want to exclude certain columns and we don't need to trim the final data frame.

I think we should keep this line in the if col_list loop.